lib/can_do.rb

```diff
@@ -38,18 +40,24 @@ def self.features
     instance.features
   end
 
+  def self.shared_features(name)
+    CONNECTION_POOL.with do |redis|
+      redis.get(FEATURE_KEY_PREFIX+name.to_s)
+    end
+  end
+
   def self.feature?(name, &block)
     name     = name.to_s
     env_name = name.upcase
 
-    fail NotAFeature.new(name) unless features.key?(name)
-
     is_enabled =
-      if ENV.key?(env_name)
+      if !shared_features(name).nil?
+        shared_features(name) == "true"
```

Comment:
you could do the same thing I did below - a regex match against `THE_TRUTH`. this would allow us to be more flexible about the *true* value.